app/models/pick.rb

```diff
@@ -0,0 +1,8 @@
+class Pick < ActiveRecord::Base
+  validates :user_id, presence: true
```

Comment:
Great! I removed the `_id` suffix from other attributes throughout the models and tests in this branch as well.